README.md

```diff
@@ -1,14 +1,51 @@
+
+# Table of Contents
+- [Features](https://github.com/instana/synthetic-synctl#features)
+- [Installation](https://github.com/instana/synthetic-synctl#installation)
+    - [Linux & Mac OS](https://github.com/instana/synthetic-synctl#linux--mac-os)
+    - [Windows](https://github.com/instana/synthetic-synctl#windows)
+- [Config an Instana Backend](https://github.com/instana/synthetic-synctl#config-an-instana-backend)
+    - [Use a config file (Recommended)](https://github.com/instana/synthetic-synctl#use-a-config-file-recommended)
+    - [Run command with --host and --token](https://github.com/instana/synthetic-synctl#run-command-with---host-and---token-no-need-to-set-a-config-file)
+    - [Use environment vars](https://github.com/instana/synthetic-synctl#use-environment-vars)
+- [Query Synthetic Test](https://github.com/instana/synthetic-synctl#query-synthetic-test)
+- [Create a synthetic test](https://github.com/instana/synthetic-synctl#create-a-synthetic-test)
+    - [Create command usage](https://github.com/instana/synthetic-synctl#create-command-usage)
+    - [Examples](https://github.com/instana/synthetic-synctl#examples)
+        - [Create an API Simple test](https://github.com/instana/synthetic-synctl#create-a-simple-ping-test)
+        - [Create an API Script test](https://github.com/instana/synthetic-synctl#create-an-api-script-test)
+        - [Create BrowserScript](https://github.com/instana/synthetic-synctl#create-browserscript)
+        - [Create WebpageScript](https://github.com/instana/synthetic-synctl#create-webpagescript)
+        - [Create WebpageAction](https://github.com/instana/synthetic-synctl#create-webpageaction)
+        - [Create Synthetic Test using json payload](https://github.com/instana/synthetic-synctl#create-synthetic-test-using-json-payload)
+- [Patch a Synthetic Test](https://github.com/instana/synthetic-synctl#patch-a-synthetic-test)
+- [Update a Synthetic Test](https://github.com/instana/synthetic-synctl#update-a-synthetic-test)
+- [Delete Synthetic test](https://github.com/instana/synthetic-synctl#delete-synthetic-test)
+- [Manage Application](https://github.com/instana/synthetic-synctl#query-application)
+    - [Get application list](https://github.com/instana/synthetic-synctl#get-application-list)
+- [Manage Synthetic Locations](https://github.com/instana/synthetic-synctl#manage-synthetic-locations)
+    - [Query synthetic location](https://github.com/instana/synthetic-synctl#query-synthetic-location)
+    - [Delete synthetic location](https://github.com/instana/synthetic-synctl#delete-synthetic-location)
+- [Manage Credentials](https://github.com/instana/synthetic-synctl#manage-credentials)
+    - [Display all credentials](https://github.com/instana/synthetic-synctl#display-all-credentials)
+    - [Create credential](https://github.com/instana/synthetic-synctl#create-credential)
+    - [Delete credential](https://github.com/instana/synthetic-synctl#delete-credential)
```

Comment:
Can we remove all the full urls here and use the #link only?  e.g  `[Delete credential](#delete-credential)`